numpy/distutils/misc_util.py

```diff
@@ -2225,6 +2224,7 @@ def is_bootstrapping():
         return True
     except AttributeError:
         return False
+        # FIXME: below code is never reached
         __NUMPY_SETUP__ = False
```

Comment:
Ok, I get this now - I think this is supposed to set `builtins.__NUMPY_SETUP__`.

Reply:
Let's leave it out, it hasn't been missed since 2009.